translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionAnyAssertion.kt

```diff
@@ -7,6 +7,8 @@ import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
  * Contains the [DescriptiveAssertion.description]s of the assertion functions which are applicable to [Any].
  */
 enum class DescriptionAnyAssertion(override val value: String) : StringBasedTranslatable {
+    TO("to"),
```

Comment:
I have divided the `Description` enums into types where they are used. This should either be placed into `DescriptionPathAssertion` or into `DescriptionBasic` if it is something we want to use in other places as well.
Maybe it makes even sense to move `TO_BE` and `NOT_TO` to DescriptionBasic - up to you if you want to do that as well

Reply:
I strongly feel that `TO` should be where `TO_BE` is. That is why I used `DescriptionAnyAssertion` in the first place.

I will move all four to `DescriptionBasic` in a separate pull request so this one keeps reviewable. I have already made the changes, the pull request is pending because I get some weird encoding issues in the tests.